### PR TITLE
Moved '.' syntax description to a separate table

### PR DESCRIPTION
**Pull request by jdgregson**
This PR addresses the poor documentation around the '.' syntax for Windows Defender Application Guard sites lists "Enterprise resource domains hosted in the cloud" and "Domains categorized as both work and personal", as discussed in issue #4408. I moved the description to its own table because it applies to both lists, and was far too long to fit into the existing table with meaningful examples.

### CODE REVIEW

**Comment by jdgregson** [comment]
I also added the text provided by @iadgovuser1 to more accurately explain how the wildcards work in practice. The existing explanation is critically incorrect, and may lead to dangerous situations where **spearphishingcontoso.com** is permitted outside of WDAG.

**Comment by iadgovuser1** [comment]
@jdgregson I like this change. IMO, it works better than trying to shove it all in the policy table.

**Comment by jdgregson** [comment]
@iadgovuser1 I agree. And even if you could find an elegant way to fit it all in there, you'd technically need to add and maintain it twice, since it applies equally to both lists.

**Comment by e0i** [comment]
cc @mypil

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md [hunk 1]
```diff
@@ -33,9 +33,17 @@ These settings, located at **Computer Configuration\Administrative Templates\Net
...
-|Domains categorized as both work and personal|At least Windows Server 2012, Windows 8, or Windows RT|A comma-separated list of domain names used as both work or personal resources. Included endpoints are rendered using Microsoft Edge and will be accessible from the Application Guard and regular Edge environment.|
- 
+|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: this list supports the wildcards detailed in the [Network isolation settings wildcards](#network-isolation-settings-wildcards) table.|
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md]:
```suggestion
|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Note: This list supports the wildcards detailed in the [Network isolation settings wildcards](#network-isolation-settings-wildcards) table.|
```

**Comment by mypil** [comment]
@jdgregson - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by jdgregson** [comment]
@mypil Review suggestions have been merged.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4408.

Thank you.

**Pull request merged**